cmd/root.go

```diff
@@ -44,7 +44,12 @@ var rootCmd = &cobra.Command{
 	Short: "Helps migration cluster configuration of a OCP 3.x cluster to OCP 4.x",
 	Long:  `Helps migration cluster configuration of a OCP 3.x cluster to OCP 4.x`,
 	Run: func(cmd *cobra.Command, args []string) {
-		env.InitConfig()
+		err := env.InitConfig()
+
+		if err != nil {
+			logrus.Fatal(err)
```

Comment:
If this is being shown to the user it would be better to tell them what may have happened. If it is going just to a log file then I would do something like:

```golang
if err != nil {
    fmt.Println("Error trying to initialize configuration, please consult the log files for the error")
    logrus.Fatal(err)
}
```

if `logrus.Fatal` writes to `stderr` then I'd want to see a more meaningful error as a user in addition to the actual error.